gammapy/irf/background.py

```diff
@@ -243,15 +251,14 @@ def __init__(
 
         if interp_kwargs is None:
             interp_kwargs = self.default_interp_kwargs
-        axes = [
-            BinnedDataAxis(
-                energy_lo, energy_hi, interpolation_mode="log", name="energy"
-            ),
-            BinnedDataAxis(
-                offset_lo, offset_hi, interpolation_mode="linear", name="offset"
-            ),
-        ]
-        self.data = NDDataArray(axes=axes, data=data, interp_kwargs=interp_kwargs)
+
+        e_edges = np.append(energy_lo, energy_hi[-1]).value * energy_lo.unit
```

Comment:
Indeed it doesn't, I have to fix this.

For me the general question was whether we want to support the case with separated `edges_mins` and `edges_maxs` on the `MapAxis` object. I think it mainly exists for "historical" reasons and is basically only used for IRFs, but I'd be OK to add it. So we could include the following:

- Add `MapAxis.edges_mins` and `MapAxis.edges_maxs` (I prefer this names slightly over `.lo` and `.hi`, because we have the `.edges` attribute and the `_min`, `_max` convention for  maps FITS I/O
- Either add a `MapAxis.from_edges(edges, edges_maxs=None)` option or add a new `MapAxis.from_edges_mins_maxs()` (which is a bit ugly name...)

Do you have any preference here @AtreyeeS @registerrier?

Reply:
For now I've decided to implement a helper function `edges_from_lo_hi`. Maybe we can re-discuss the question whether to add this functionality to `MapAxis` in a different PR. I'll make reminder issue.

Reply:
See #2134.